test/e2e/lib/driver-helper.js

```diff
@@ -320,36 +327,11 @@ export function checkForConsoleErrors( driver ) {
 	}
 }
 
-export function printConsole( driver ) {
-	if ( config.get( 'printConsoleLogs' ) === true ) {
```

Comment:
`printConsoleLogs` was never set in CI, and I think it wouldn't have worked locally because Magellan hides the output from the tests.

Anyway, we have the logs in a file now, no need for this flag.